explore/src/main/scala/explore/Explore.scala

```diff
@@ -121,7 +124,15 @@ object ExploreMain extends IOApp.Simple {
 
     Dispatcher[IO].allocated
       .map(_._1)
-      .flatMap { implicit dispatcher =>
+      .flatMap { d =>
+        for {
+          p <- ExploreLocalPreferences.loadPreferences[IO]
+          l <- setupLogger(p)
+        } yield (d, l, p)
+      }
+      .flatMap { param =>
+        implicit val (dispatcher, logger, _)                     = param
+        implicit val localPreferences                            = param._3
```

Comment:
Can `localPreferences` be obtained from the unapply in the previous line?

Reply:
It can but I don't need it to be implicit, though now that I look at it again it is `implicit` anyway